lte/gateway/python/magma/pipelined/app/classifier.py

```diff
@@ -283,21 +293,22 @@ def _discard_tunnel_flows(self, precedence:int, i_teid:int,
                        priority=priority + 1)
 
         # discard downlink Tunnel for LOCAL port
-        match = MagmaMatch(eth_type=ether_types.ETH_TYPE_IP,
-                           in_port=self._uplink_port, ipv4_dst=ue_ip_adr)
+        ip_match_out = get_ue_ip_match_args(ue_ip_adr, Direction.IN)
+        match = MagmaMatch(eth_type=get_eth_type(ue_ip_adr),
+                           in_port=self._uplink_port, **ip_match_out)
 
         flows.add_flow(self._datapath, self.tbl_num, match,
                        priority=priority + 1)
 
         # discard downlink Tunnel for mtr port
-        match = MagmaMatch(eth_type=ether_types.ETH_TYPE_IP,
-                           in_port=self.config.mtr_port,ipv4_dst=ue_ip_adr)
- 
+        match = MagmaMatch(eth_type=get_eth_type(ue_ip_adr),
+                           in_port=self.config.mtr_port, **ip_match_out)
+
         flows.add_flow(self._datapath, self.tbl_num, match,
```

Comment:
should this function be calling delete_flow?

Reply:
otherwise I'm not sure what its doing

Reply:
As per OAI code in GTPApplication.cpp for discard_downlink_tunnel_flow, using OFPFC_ADD with cookie values.